src/Command/Command.php

```diff
@@ -42,4 +47,26 @@ public function getIo()
     {
         return $this->io;
     }
+
+    /**
+     * Get maintenance mode.
+     *
+     * @return bool
+     */
+    public function getMaintenance()
```

Comment:
@LOBsTerr Since is `boolean` can we rename method `getMaintenance` as `isMaintenance`.

Reply:
@jmolivas I have fixed it